identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -70,9 +70,20 @@ <h2 class="form__heading">Newsletter options</h2>
 
     </div>
 
-    <div class="manage-account__modal manage-account__modal--newsletterFormat">
+    <fieldset class="fieldset fieldset--manage-account-rightsubmit js-manage-account__ajaxForm-submit">
+        <div class="fieldset__heading"></div>
```

Comment:
Use `<h1> - <h6>` instead of the `<div>`?

Reply:
i'm unhappy with the whole empty div fieldsets need in general but this is on pasteup, defo for another pr :/